app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation.scss

```diff
@@ -0,0 +1,107 @@
+.gem-c-super-navigation-header__logotype-crown {
+  top: -2px;
+}
+
+.gem-c-super-navigation-header {
+  background: govuk-colour("black");
+}
+
+.gem-c-super-navigation-header__header-logo {
+  padding-top: 13px;
+  padding-bottom: 14px;
+
+  @include govuk-media-query($from: "desktop") {
+    display: block;
+    float: left;
+  }
+}
+
+.gem-c-super-navigation-header__content {
+  @include govuk-media-query($from: "desktop") {
+    display: block;
+    float: right;
+  }
+}
+
+.gem-c-super-navigation-header__items {
+  list-style: none;
+  margin: 0;
+  padding: 0 0 govuk-spacing(1) 0;
+
+  @include govuk-media-query($from: "desktop") {
+    padding: 0;
+  }
+}
+
+.gem-c-super-navigation-header__item {
+  margin-right: 0;
+  padding: govuk-spacing(1) 0;
+
+  @include govuk-media-query($from: "desktop") {
+    float: left;
+    padding: 0;
+  }
+}
+
+.gem-c-super-navigation-header__item-link {
+  display: block;
+  padding: govuk-spacing(3) 0;
+
+  @include govuk-media-query($from: "desktop") {
+    padding: govuk-spacing(4);
+    width: auto;
+  }
+}
+
+// Special search link
+.gem-c-super-navigation-header__item--search {
+  border-bottom: 0;
+}
+
+.gem-c-super-navigation-header__item-link--search {
+  @include govuk-media-query($from: "desktop") {
+    background-color: $govuk-link-colour;
+    background-image: image-url("govuk_publishing_components/search-icon.svg");
```

Comment:
I'd suggest inlining the SVG because background images can be removed or obscured by a 'readability backplate' in forced colour modes.
 
For example, the search icon currently disappears in Edge with Windows High Contrast Mode:
![Screenshot 2021-07-02 at 09 19 33](https://user-images.githubusercontent.com/121939/124244227-9f1a5a80-db16-11eb-81f3-84bf37d83236.png)

And also in Firefox with custom colours set:
![Screenshot 2021-07-02 at 09 18 45](https://user-images.githubusercontent.com/121939/124244117-814cf580-db16-11eb-82d0-5ea05e4847ce.png)

This will also allow you to use `currentColor` for the stroke colour, and the SVG should then inherit the text colour, which will make the focus state change easier to manage, and also respect user's colour preferences in forced colour mode – see the crown for an example of how this works.

Reply:
Good idea - thanks for spotting this Ollie; I've updated and changed the magnifying glass icon to be inline and use currentColor.